task1/task1.js

```diff
@@ -0,0 +1,86 @@
+// 1. Write a JavaScript function that takes two numbers as parameters and returns their sum.
+let func1 = (a, b) => a + b;
```

Comment:
If any function or variable is just defined, and there is no intention re-assign it - use `const` instead of `let`. It's kind of "best practice"